SwiftCodeBattleHelper/ContentView.swift

```diff
@@ -111,8 +112,11 @@ struct ContentView: View {
                         Text("stdin")
                         pasteButton
                         deleteButton
+                        Spacer()
+                        countText
```

Comment:
🎨 `countText` should be positioned at the bottom-right of the code pane.
Would you prefer to take care of it, or would you like me to handle it from here?

Reply:
Adjusted the position, but not sure if it's the style you want.
I don't have much design preferences in this app, you're free to adjust the UI.